src/main/java/com/arsenarsen/lavaplayerbridge/utils/JDAMultiShard.java

```diff
@@ -1,24 +1,25 @@
 package com.arsenarsen.lavaplayerbridge.utils;
 
+import net.dv8tion.jda.bot.sharding.ShardManager;
+import net.dv8tion.jda.bot.utils.cache.ShardCacheView;
 import net.dv8tion.jda.core.JDA;
 import net.dv8tion.jda.core.entities.Guild;
 
 import java.util.Arrays;
 import java.util.Objects;
 
 public class JDAMultiShard {
-    private JDA[] jdas;
+    private ShardManager manager;
 
-    public JDAMultiShard(JDA[] jdas){
-        this.jdas = jdas;
+    public JDAMultiShard(ShardManager manager){
+        this.manager = manager;
     }
 
     public boolean isValid(String id){
         return getGuild(id) != null;
```

Comment:
Just replace this line with manager.getGuildById there's no point calling another method for